client/packages/system/src/Patient/DefaultPatientUISchema.json

```diff
@@ -6,9 +6,28 @@
       "label": "Patient Information",
       "elements": [
         {
-          "type": "Control",
+          "type": "IdGenerator",
+          "label": "Code",
           "scope": "#/properties/code",
-          "label": "Code"
+          "options": {
+            "parts": [
+              {
+                "type": "StoreName",
+                "mutations": [{ "firstNChars": 3 }, { "toUpperCase": true }]
+              },
+              {
+                "type": "Number",
+                "numberName": "PatientCode",
+                "mutations": [{ "padString": "0000" }]
+              }
+            ],
+            "allowManualEntry": true,
+            "validations": [
+              {
+                "type": "UniquePatientCode"
+              }
+            ]
+          }
```

Comment:
This is the new default add patient form (as discussed, not implemented to pulled from a local database)